[CSS Modules] Adding "one and done" suggestions to Open Issues (#1146)
MicrosoftEdge/MSEdgeExplainers: ShadowDOM/explainer.md

> * Should the stylesheet in `<style type="module">` be accessible via JavaScript? In regular `<style>` tags, it would be accessible through the [LinkStyle interface](https://www.w3.org/TR/cssom-1/#the-linkstyle-interface), and thus the stylesheet can be modified through this interface. We may not want to allow this, particularly if the CSS Module approach is decided to be "one and done".

Reviewer: ```suggestion
* Should the stylesheet for a declarative CSS module be accessible via JavaScript from the `<style type="module">` element? In regular `<style>` tags, it would be accessible and modifiable through the [LinkStyle interface](https://www.w3.org/TR/cssom-1/#the-linkstyle-interface). We may not want to allow this, particularly if the CSS Module approach is decided to be "one and done".
```
Suggested a rephrase to avoid potential confusion, since the stylesheet in a `<style type="module">` *will* be accessible via JavaScript from `shadow.adoptedStyleSheets`. It's just getting it from the `<style>` element that we're considering restricting.
